### DISCUSSION TITLE
Failed to install dependencies for pubspec file in /Users/builder/clone.

### DISCUSSION THREAD

**Discussion by rignaneseleo** [codemagic-ci-cd/codemagic-docs#1634] [Q&A]
Hi, I'm trying to compile a Flutter app from this GitHub link https://github.com/rignaneseleo/SlimSocial-for-Facebook

I get this error after a few seconds:
```
== Install Flutter dependencies ==

Build failed :|
Failed to install dependencies for pubspec file in /Users/builder/clone. Directory was not found
```

Ihave no idea why, I've setup many projects and only this one has this problem.

Build id is `64596c0cebf4681f9177351e`

**Comment by dtrdic** [comment]
Hey Leo, the error message suggests that Codemagic is not able to locate the directory where your Flutter project is located. Make sure that you have the correct path to your Flutter project.
I you are using workflow editor, go into workflow settings, and under the build section there should be a project path dropdown. Select your project from the dropdown.
Try to build again

**Comment by rignaneseleo** [comment]
It worked, but it was tricky because it didn't show the folder in the dropdown until I pressed on the refresh button.
